src/Common/Types.ts

```diff
@@ -396,12 +363,48 @@ interface ApprovalUserDataType {
     userGroups?: Pick<UserGroupDTO, 'identifier' | 'name'>[]
 }
 
+export interface UserApprovalInfo {
+    requiredCount: number
+    currentCount: number
+    approverList: {
+        hasApproved: boolean
+        canApprove: boolean
+        identifier: string
+    }[]
+}
+
+export enum ApprovalConfigDataKindType {
```

Comment:
Please check with BE if we can extend existing enum of ConfigResourceType which is being used across protection flow